packages/theme/src/core/table/types.ts

```diff
@@ -2,9 +2,11 @@ import { FontVariants } from '../font/types';
 
 export interface TableTheme {
     borderColor: string;
+    borderRadius?: string;
     scrollBarColor: string;
     shadowColor: string;
     row: {
+        fontVariant?: FontVariants;
```

Comment:
Could you please add default values of these types in the `index` file also.

Reply:
updated accordingly.